upstream-community-operators/poison-pill-operator/0.0.1/bundle.Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM scratch
```

Comment:
You can remove this file. It will be ignored.

Reply:
thanks. removed